components/admin_console/admin_definition.jsx

```diff
@@ -4476,7 +4476,7 @@ const AdminDefinition = {
                                 display_name: t('admin.openid.google'),
                                 display_name_default: 'Google Apps',
                                 help_text: t('admin.google.EnableMarkdownDesc'),
```

Comment:
Hey Scott!

You may want to rename these ID's for all the OpenID since they will conflict with the OAUTH. For example, `admin.google.EnableMarkdownDesc` is also used for OAuth. It may be better to rename this to `admin.google.openid.EnableMarkdownDesc` to avoid the conflict. This is also true for all the providers which I just noticed. It could be a contributing factor to the above problem mentioned by Malik.

Reply:
This was actually intentional since the steps for both should be the same.